config/settings/base.py

```diff
@@ -609,9 +610,29 @@
     },
 }
 
-# Time estimates for reviewing tasks based on the assigned story points
-#
-# SPRINT_HOURS_RESERVED_FOR_REVIEW {
+
+def json_keys_to_float(json_dict):
+    if isinstance(json_dict, dict):
+        processed_json_dict = {}
+        for key, value in json_dict.items():
+            processed_key: Optional[float]
+            try:
+                processed_key = float(key)
+            except ValueError:
+                processed_key = None
+            processed_json_dict[processed_key] = value
+        return processed_json_dict
+    return json_dict
+
+
+def preprocess_sprints_hours_reserved_for_review(json_dict):
+    if "null" not in json_dict:
+        raise ValueError('required "null" key is missing for SPRINT_HOURS_RESERVED_FOR_REVIEW')
```

Comment:
This should raise [ImproperlyConfigured](https://docs.djangoproject.com/en/3.2/ref/exceptions/#improperlyconfigured).
```suggestion
        raise ImproperlyConfigured('Required "null" key is missing from "SPRINT_HOURS_RESERVED_FOR_REVIEW".')
```